### PR TITLE
feat(classification): New classification badge design

### PR DESCRIPTION
**Pull request by patrickpaul**
Moves Classification labels to the new design with a single color and larger border radius.

### CODE REVIEW

File: src/components/time/__tests__/__snapshots__/ReadableTime.test.js.snap [hunk 1]
```diff
@@ -1,10 +1,10 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`components/time/ReadableTime should use default relative threshold if not provided 1`] = `"in 1,800,000 seconds"`;
```
Comment (booc0mtaco) [inline src/components/time/__tests__/__snapshots__/ReadableTime.test.js.snap:3]: this should probably not be changing. double check your node version; for buie it should use >=12.x 

cc @alexkrolick

File: src/features/security/_mixins.scss [hunk 2]
```diff
@@ -4,13 +4,12 @@
...
-    padding: 0 5px;
-    border: 1px solid;
-    border-radius: $bdl-border-radius-size;
+    padding: 2px 8px;
+    border-radius: $bdl-border-radius-large-size;
 }
 
 @mixin bdl-SecurityBadge-name {
-    margin-left: 5px;
+    margin-left: 4px;
```
Comment (booc0mtaco) [inline src/features/security/_mixins.scss:12]: we now have `$bdl-grid-unit` (4px) for margins/paddings. let's use where possible

File: src/icons/general/IconSecurityClassification.tsx [hunk 3]
```diff
@@ -5,31 +5,15 @@ import AccessibleSVG from '../accessible-svg';
...
 
-const IconSecurityClassification = ({
-    className = '',
-    height = 32,
-    color = bdlGray,
-    title,
-    strokeWidth = 2,
-    width = 32,
-}: Icon) => {
+const IconSecurityClassification = ({ className = '', height = 12, color = bdlGray, title, width = 12 }: Icon) => {
```
Comment (booc0mtaco) [inline src/icons/general/IconSecurityClassification.tsx:8]: can we put a comment here to eventually move to the asset repo version of this? Its a small bit of refactor work so OK to come back to

is it this icon more or less? http://opensource.box.com/box-ui-elements/storybook/?path=/story/icon-line-shield16--shield-16

File: src/components/time/__tests__/__snapshots__/ReadableTime.test.js.snap (hunk 1 above)
Comment (patrickpaul) [reply src/components/time/__tests__/__snapshots__/ReadableTime.test.js.snap]: Resolved.

**Pull request merged**